README.md

```diff
@@ -63,510 +60,95 @@ and create the secret via `kubectl create secret tls ${CERT_NAME} --key ${KEY_FI
+3. [Custom template](docs/user-guide/custom-template.md): when more specific settings are required, like [open_file_cache](http://nginx.org/en/docs/http/ngx_http_core_module.html#open_file_cache), adjust [listen](http://nginx.org/en/docs/http/ngx_http_core_module.html#listen) options as `rcvbuf` or when is not possible to change the configuration through the ConfigMap.
 
 ## Source IP address
 
-By default NGINX uses the content of the header `X-Forwarded-For` as the source of truth to get information about the client IP address. This works without issues in L7 **if we configure the setting `proxy-real-ip-cidr`** with the correct information of the IP/network address of the external load balancer.
-If the ingress controller is running in AWS we need to use the VPC IPv4 CIDR. This allows NGINX to avoid the spoofing of the header.
-Another option is to enable proxy protocol using `use-proxy-protocol: "true"`.
-In this mode NGINX do not uses the content of the header to get the source IP address of the connection.
-
-## Exposing TCP services
-
-Ingress does not support TCP services (yet). For this reason this Ingress controller uses the flag `--tcp-services-configmap` to point to an existing config map where the key is the external port to use and the value is `<namespace/service name>:<service port>:[PROXY]:[PROXY]`
-It is possible to use a number or the name of the port. The two last fields are optional. Adding `PROXY` in either or both of the two last fields we can use Proxy Protocol decoding (listen) and/or encoding (proxy_pass) in a TCP service (https://www.nginx.com/resources/admin-guide/proxy-protocol/).
-
-The next example shows how to expose the service `example-go` running in the namespace `default` in the port `8080` using the port `9000`
-
-```yaml
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: tcp-configmap-example
-data:
-  9000: "default/example-go:8080"
-```
-
-Please check the [tcp services](examples/tcp/README.md) example
-
-## Exposing UDP services
-
-Since 1.9.13 NGINX provides [UDP Load Balancing](https://www.nginx.com/blog/announcing-udp-load-balancing/).
-
-Ingress does not support UDP services (yet). For this reason this Ingress controller uses the flag `--udp-services-configmap` to point to an existing config map where the key is the external port to use and the value is `<namespace/service name>:<service port>`
-It is possible to use a number or the name of the port.
-
-The next example shows how to expose the service `kube-dns` running in the namespace `kube-system` in the port `53` using the port `53`
+By default NGINX uses the content of the header `X-Forwarded-For` as the source of truth to get information about the client IP address. This works without issues in L7 **if we configure the setting `proxy-real-ip-cidr`** with the correct information of the IP/network address of trusted external load balancer.
```

Comment:
So, should we make explicit how to disable this behaviour? Something like 'not setting proxy-real-ip-cidr will disable this behaviour? Or setting this as 127.0.0.1 is the right config to disable?